local-storage/src/LocalStorage.ts

```diff
@@ -0,0 +1,97 @@
+import { CurriedFunction2, curry2 } from '@most/prelude';
+import {
+  LocalStorageClearCommand,
+  LocalStorageCommand,
+  LocalStorageRemoveItemCommand,
+  LocalStorageSetItemCommand,
+  LocalStorageSource,
+} from './types';
+import {
+  Stream,
+  constant,
+  drain,
+  filter,
+  fromEvent,
+  map,
+  merge,
+  startWith,
+} from 'most';
+
+import { hold } from '@most/hold';
+
+export interface LocalStorageSinks {
+  localStorage$: Stream<LocalStorageCommand>;
+}
+
+export interface LocalStorageSources {
+  localStorage: LocalStorageSource;
+}
+
+export function LocalStorage(sinks: LocalStorageSinks): LocalStorageSources {
```

Comment:
I like the possibility of function composition for these things now. We can now also write higher-order-effectful-components just as we have been writing HOCs for our main functions and components for some time now. There is a bit of beautiful symmetry, if I do say so myself :smile: